1003/bn.md

```diff
@@ -0,0 +1,88 @@
+# Light OJ - 1003 : Drunk
+---
+
+### প্রবলেম সারমর্ম  
+---
+এই প্রবলেমে,
+- তোমার কাজ হলো মাতাল হওয়া সম্ভব কিনা বের করা।
+- মাতাল হবার জন্য **m** টি পূর্বশর্ত পূরণ করতে হবে।
+- প্রতিটি পূর্বশর্তে, তোমাকে ২ টি ড্রিঙ্ক **a** এবং **b** দেয়া হবে।  মাতাল হবার জন্য তোমাকে **b** এর আগে **a** পান করতে হবে।
+- এখন মাতাল হবার জন্য তোমাকে অবশ্যই সব পূর্বশর্ত মেনে সব ড্রিঙ্ক পান করতে হবে। 
+
+### হিন্ট
+---
+কোন কোন কেইস এ তুমি মাতাল হতে পারবে না এটা চিন্তা করো।
+
+### মূল ধারণা
+---
+পূর্বশর্ত যদি বলে:
+```html
+তোমার <soda>  এর আগে  <wine>
+     <wine> এর আগে  <water> এবং
+     <water>  এর আগে  <soda> খেতে হবে।
+```
+<img src="https://github.com/codermehraj/problem-tutorials/blob/main/1003/relation.png?raw=true" width="35%" height="30%"> <br>
```

Comment:
just use 

`![relation image](relation.png)`

same for en.md

Reply:
I have changed to `![relation image](relation.png)`